x-pack/plugins/apm/public/components/fleet_integration/apm_agents/index.tsx

```diff
@@ -0,0 +1,326 @@
+import {
+  createDotNetAgentInstructions,
+  createDjangoAgentInstructions,
+  createFlaskAgentInstructions,
+  createGoAgentInstructions,
+  createJavaAgentInstructions,
+  createJsAgentInstructions,
+  createNodeAgentInstructions,
+  createPhpAgentInstructions,
+  createRailsAgentInstructions,
+  createRackAgentInstructions,
+} from '../../../../server/tutorial/instructions/apm_agent_instructions';
+import { AgentName } from '../../../../typings/es_schemas/ui/fields/agent';
+import { AgentIcon } from '../../shared/agent_icon';
+import {
+  NewPackagePolicy,
+  PackagePolicy,
+  PackagePolicyEditExtensionComponentProps,
+} from '../apm_policy_form/typings';
+import { getCommands } from '../../../tutorial/config_agent/commands/get_commands';
+import { CopyCommands } from '../../../tutorial/config_agent/copy_commands';
+// import { replaceTemplateStrings } from '../../../../../../../src/plugins/home/public/application/components/tutorial/replace_template_strings.js';
+
+interface Props {
+  policy: PackagePolicy;
+  newPolicy: NewPackagePolicy;
+  onChange: PackagePolicyEditExtensionComponentProps['onChange'];
+}
+
+function AccordianButtonContent({
+  agentName,
+  title,
+}: {
+  agentName: AgentName;
+  title: string;
+}) {
+  return (
+    <EuiFlexGroup justifyContent="flexStart" alignItems="center">
+      <EuiFlexItem grow={false}>
+        <AgentIcon size="xl" agentName={agentName} />
+      </EuiFlexItem>
+      <EuiFlexItem grow={1}>
+        <EuiFlexGroup direction="column" gutterSize="none">
+          <EuiFlexItem>
+            <EuiText>
+              <h4>{title}</h4>
+            </EuiText>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiText size="s">
+              <p>Configure instrumentation for {title} applications.</p>
+            </EuiText>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFlexItem>
+    </EuiFlexGroup>
+  );
+}
+
+export function ApmAgents({ newPolicy, onChange }: Props) {
+  const ELASTIC_WEBSITE_URL =
+    useKibana().services.docLinks?.ELASTIC_WEBSITE_URL ?? '';
+  return (
+    <div>
+      {ApmAgentInstructions.map(
+        ({ agentName, title, createAgentInstructions, variantId }) => (
+          <>
+            <EuiPanel>
+              <EuiAccordion
+                id={agentName}
+                buttonContent={
+                  <AccordianButtonContent agentName={agentName} title={title} />
+                }
+              >
+                <AgentInstrumentationInstructions
+                  newPolicy={newPolicy}
+                  createAgentInstructions={createAgentInstructions}
+                  elasticWebsiteUrl={ELASTIC_WEBSITE_URL}
+                  variantId={variantId}
+                />
+              </EuiAccordion>
+            </EuiPanel>
+            <EuiSpacer />
+          </>
+        )
+      )}
+    </div>
+  );
+}
+
+function Content({ text }: { text: string }) {
+  const whiteListedRules = ['backticks', 'emphasis', 'link', 'list'];
```

Comment:
Maybe call it `allowedListedRules`?

Reply:
i'm not even sure if it's necessary to list these for our content...i just copy-pasted this from the home/tutorial plugin